test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -2430,6 +2430,7 @@ static void traffic_mngr_test_shaper_profile(void)
 		shaper_params.peak_rate    = clamp_rate(idx * MIN_PEAK_BW);
 		shaper_params.commit_burst = clamp_burst(idx * MIN_COMMIT_BURST);
 		shaper_params.peak_burst   = clamp_burst(idx * MIN_PEAK_BURST);
+		shaper_params.dual_rate    = true;
```

Comment:
There is no need to add this line here. The dual_rate field is already set before this loop. The existing assignment
should simply be modified to set the field to true instead of false.

Then I would like to improve the commit message of this commit too. e.g.:
```
validation: tm: set dual_rate flag in shaper profile test

Set dual_rate to true in the shaper profile test which checks
that shaper profile can be created with the desired parameters,
including peak_rate and peak_burst.
```